src/coreclr/src/pal/src/file/path.cpp

```diff
@@ -616,79 +613,24 @@ FILEDosToUnixPathA(
             *p = '/';
         }
 
-        if (pPointAtDot)
+        if (*p == '*')
```

Comment:
I would think we should delete this normalization of `*.*` to `*` too. The managed libraries do not have normalization like this, so the PAL should not have it either.